bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java

```diff
@@ -462,6 +462,9 @@ public Type getType() {
     @Override
     @Nullable
     public String getValidationExpression() {
+        if ("".equals(validationExpression)) {
```

Comment:
You could use the jackson annotation at interface, `@JsonInclude(JsonInclude.Include.NON_EMPTY)` on getValidationExpression to only include non empty values